srcpkgs/limine/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'limine'
+pkgname=limine
+version=2.24
+revision=1
+archs="x86_64"
```

Comment:
Your short_desc says
> Advanced x86/x86_64 BIOS and UEFI bootloader
so `archs="x86_64* i686*"` sounds more appropriate
If it is broken for some reaoson on `*musl` or `i686` than document the reason (and set a apporpiate `broken=` statement in the template

Reply:
On i686 it would need to either disable the UEFI component or use a 64 bit cross compiler, any suggestion? And it works on musl just fine.

Reply:
Nah, using a 64bit cross on i686 is messy. If you can get it easly to only build the 32bit bios component, than do that. Otherwise just keep it `x86_64*`

Reply:
Will do.